SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -147,39 +166,23 @@ if [ ! -f "${REQ_PATH}" ]; then
     exit 1
 fi
 
+# Check if an environment already exists in the session, avoiding multiple environments
 CURRENT_ENV_NAME=$(find "/home/$USER" -type d -name "*_env" | head -n 1 | cut -d '/' -f4)
 CURRENT_REPO_PATH=$(tail -n 1 "/home/$USER/.bash_profile" | cut -d ' ' -f2)
-
-# Check if an environment already exists in the session
 if [ -n "${CURRENT_ENV_NAME}" ]; then
     _log "ENVIRONMENT_ALREADY_EXISTS:${CURRENT_ENV_NAME}"
     _log "REPO_PATH:${CURRENT_REPO_PATH#$HOME}"
     exit 1
 fi
 
-# Create environment (acc-py or generic)
-if [ -n "$ACCPY_VERSION" ]; then
-    source $ACCPY_PATH/base/${ACCPY_VERSION}/setup.sh
-    acc-py venv ${ENV_PATH} | tee -a "${LOG_FILE}"
-else
-    _log "Creating environment ${ENV_NAME} using Generic Python..."
-    python -m venv ${ENV_PATH} | tee -a "${LOG_FILE}"
-fi
+_log "Creating environment ${ENV_NAME} using ${BUILDER} (${BUILDER_VERSION})..."
+source "${BUILDER_PATH}"
 
-_log "ENV_NAME:${ENV_NAME}"
+$(echo ${ACTIVATE_ENV_CMD})
```

Comment:
Is this echo a debug message that we needed to remove?

Reply:
the `ENV_NAME` log was moved down and that echo is the way I found for making sure it is activated no matter the command specified in the builder scripts

Reply:
Ah ok it's an execution, never mind.

Reply:
I know it may be redundant because is activated already in the builder scripts, its just a preventive measure. If not needed, it'll be removed in future PRs
The `ACTIVATE_ENV_CMD` is also useful for activating the environment everytime the terminal is opened in the lab.

Reply:
Ah true, why the need to activate again?